CHANGELOG

```diff
@@ -51,6 +51,7 @@ Qtile X.X.X, released XXXX-XX-XX:
         - switch to Github Actions for CI
         - Columns layout has new `margin_on_single` option to specify margin
           size when there is only one window (default -1: use `margin` option).
+        - new format attribute for WindowName widget
```

Comment:
Can you add the 'max_chars' option to this too? It's a pretty nice addition in its own right!